apps/sms/services/js/service_mixin.js

```diff
@@ -0,0 +1,56 @@
+/*global bridge */
+
+'use strict';
+
+if (!('bridge' in self)) {
+  importScripts('/lib/bridge.js');
+}
+
+(function(exports) {
+
+  /**
+   * This mixin allows to expose easily a service from an object.
+   *
+   * This needs the following properties to work properly:
+   *
+   * - SERVICE_NAME holds the service name.
+   * - METHODS holds the methods that will be exposed by the service.
+   * - STREAMS holds the methods that handle streams exposed by the service.
+   *
+   * EVENTS, METHODS and STREAMS are all optional.
+   */
+  var serviceMixin = {
+    initService() {
+      var service = bridge.service(this.SERVICE_NAME);
+
+      if (this.METHODS && this.METHODS.length) {
+        this.METHODS.forEach((exposedMethod) => {
+          service.method(exposedMethod, this[exposedMethod]);
+        });
+      }
+
+      if (this.STREAMS && this.STREAMS.length) {
+        this.STREAMS.forEach((exposedStream) => {
+          service.stream(exposedStream, this[exposedStream]);
+        });
+      }
+
+      return service;
+    }
+  };
+
+  exports.ServiceMixin = {
```

Comment:
Wondering if we need to mention ```bridge``` somewhere in the naming of mixer or mixer methods?